nvi-commons/src/test/java/no/sikt/nva/nvi/common/service/NviServiceTest.java

```diff
@@ -483,40 +505,53 @@ public void shouldWriteVersionOnRefreshWhenStartMarkerIsNotSet() {
 
     @Test
     public void refreshVersionShouldContinue() {
-        candidateRepository.create(randomCandidate(), List.of());
-        candidateRepository.create(randomCandidate(), List.of());
-        candidateRepository.create(randomCandidate(), List.of());
+        IntStream.range(0, 3).forEach(i -> candidateRepository.create(randomCandidate(), List.of()));
+
         var result = nviService.refresh(1, null);
         assertThat(result.shouldContinueScan(), is(equalTo(true)));
     }
 
     @Test
     public void shouldWriteVersionOnRefreshWithStartMarker() {
-        candidateRepository.create(randomCandidate(), List.of());
-        candidateRepository.create(randomCandidate(), List.of());
+        IntStream.range(0, 2).forEach(i -> candidateRepository.create(randomCandidate(), List.of()));
 
-        // Read candidates with scan to know the order for later comparison
-        var candidates =
-            localDynamo.scan(ScanRequest.builder().tableName(ApplicationConstants.NVI_TABLE_NAME).build())
-                .items()
-                .stream()
-                .filter(a -> a.get("type").s().equals("CANDIDATE")).toList();
+        var candidates = getCandidatesInOrder();
 
-        var identifierForFirstRow = UUID.fromString(candidates.get(0).get("identifier").s());
-        var identifierForSecondRow = UUID.fromString(candidates.get(1).get("identifier").s());
+        List<CandidateDao> originalRows = getCandidateDaos(candidates);
```

Comment:
var

Reply:
btw just found this nice setting :-)
![image](https://github.com/BIBSYSDEV/nva-nvi/assets/554713/06e737c4-c1d4-4099-8882-cb0289f7d354)